.github/workflows/deploy.yml

```diff
@@ -78,27 +78,36 @@ jobs:
           APP_SECRETS:   ${{ secrets[format('APP_SECRETS_{0}', env.DEPLOY_ENV)] }}
           INFRA_SECRETS: ${{ secrets[format('INFRA_SECRETS_{0}', env.DEPLOY_ENV)] }}
 
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets[format('AZURE_CREDENTIALS_{0}', env.DEPLOY_ENV)] }}
+
+      - name: Validate Azure key vault secrets
```

Comment:
can you use action from https://github.com/DFE-Digital/github-actions/tree/master/validate-key-vault-secrets
usage https://github.com/DFE-Digital/find-teacher-training/compare/master..103b3054fca7523d14511bc3c8558aa9647ce061#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R66-R91